.github/workflows/remote-cd-trigger-template.yml

```diff
@@ -0,0 +1,45 @@
+name: Template - Remote trigger to CDCent
+on:
+  workflow_call:
+    inputs:
+      WORKFLOW:
+        type: string
+        description: "Workflow yml file that should be triggered."
+        required: true
+      REF: 
+        type: string
+        description: "Git tag or branch the workflow is running on."
+        required: false
+        default: 'main'
+
+jobs:
+  invoke-cd-trigger:
```

Comment:
Seems like there is a step missing to build and push the image to CDC ImageHub.  The step to do both the build and publish to ImageHub is simply `gradle jib`.  All that is required is the two ENV vars set for the image hub username and password.

Reply:
adding the imagehub environment variables at cdc gov and cdc ent repos and will see how this gradle jib works. was not sure how to include this at moment..